packages/common/src/ui/components/menus/DropdownMenu/DropdownMenu.tsx

```diff
@@ -80,25 +80,27 @@ export const DropdownMenu: FC<DropdownMenuProps> = ({
   children,
   disabled = false,
 }) => {
-  return (
-    <FormControl size="small">
-      <InputLabel
-        shrink={false}
-        sx={{ color: '#8f90a6', '&.Mui-focused': { color: '#8f90a6' } }}
-        id={`action-drop-down-label-${label}`}
-      >
-        {label}
-      </InputLabel>
-      <StyledSelect
-        disabled={disabled}
-        value=""
-        size="small"
-        labelId={`action-drop-down-label-${label}`}
-        variant="outlined"
-        IconComponent={ChevronDownIcon}
-      >
-        {children}
-      </StyledSelect>
-    </FormControl>
-  );
+  if (Array.isArray(children))
+    return (
+      <FormControl size="small">
+        <InputLabel
+          shrink={false}
+          sx={{ color: '#8f90a6', '&.Mui-focused': { color: '#8f90a6' } }}
+          id={`action-drop-down-label-${label}`}
+        >
+          {label}
+        </InputLabel>
+        <StyledSelect
+          disabled={disabled}
+          value=""
+          size="small"
+          labelId={`action-drop-down-label-${label}`}
+          variant="outlined"
+          IconComponent={ChevronDownIcon}
+        >
+          {children}
+        </StyledSelect>
+      </FormControl>
+    );
+  else return <div>{children}</div>;
```

Comment:
Not sure about this -- might be better to leave it as it was. It's probably a bit risky to have a "Delete" button so easily accessible, at least when it's inside a Dropdown you have to be a little more intentional, therefore less likely to accidentally click it.

Reply:
possibly, given that the action in almost all cases is 'delete'. It's also more consistent having the drop down everywhere.

I really like the ease of having the button available without the select.. but am worrying that you are right. Also.. didn't want to abandon all this work! 

Am favouring reverting back to the select.. but could go either way.. thoughts?

Reply:
Yeah, let's leave it how it was until we can be bothered adding a Warning modal when deleting stuff. Now that I've done this, it's an easy change to put back when we need it.

Reply:
Reverted 97b222e9
Made new issue: #1115